data/json/furniture_and_terrain/appliances.json

```diff
@@ -32,6 +32,40 @@
     ],
     "damage_reduction": { "all": 32 }
   },
+  {
+    "id": "ap_oven",
+    "name": { "str": "oven" },
+    "description": "A standard electric convection-based oven, commonly used for heating and cooking food.",
+    "symbol": "#",
+    "color": "dark_gray",
+    "broken_color": "light_cyan",
+    "broken_symbol": "x",
+    "categories": [ "utility" ],
+    "damage_modifier": 10,
+    "damage_reduction": { "all": 30 },
+    "durability": 80,
+    "flags": [ "CARGO", "OBSTACLE", "COVERED", "FLAT_SURF" ],
+    "pseudo_tools": [ { "id": "hotplate", "hotkey": "h" } ],
+    "epower": 0,
```

Comment:
I think 0 here means it doesn't consume power

Reply:
I was going to test that, it shouldn't consume power unless you're using it as a hotplate.

Reply:
Right, then my suggestion to add CTRL_ELECTRONIC flag might have been a mistake if it works as a tool you don't need to turn it on

Reply:
I'll see what happens and if it is making sense at the moment

Reply:
It did not, btw.